packages-ts/gauntlet-cosmos-contracts/src/commands/contracts/proxy_ocr2/confirmContract.ts

```diff
@@ -13,10 +13,8 @@ type ContractInput = {
 }
 
 const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInput> => {
-  const contract = args[0]
```

Comment:
this was a bug because the arguments to the confirm and propose method are not the proxy contract address but the contract address